docs/manifest.json

```diff
@@ -695,6 +695,12 @@
 		"markdown_source": "../packages/components/src/date-time/README.md",
 		"parent": "components"
 	},
+	{
+		"title": "DetectFocusOutside",
```

Comment:
This seems useful for the Gallery block.  I know we merged a similar thing to unselect images lately cc @nosolosw

Reply:
Oh, nice. Yeah, it looks like the same use case ([link](https://github.com/WordPress/gutenberg/pull/14930#discussion_r348383465) for reference). Note that I didn't extract this logic to a reusable component there, so we can refactor gallery to use this when the PR lands.

Reply:
@nosolosw FWIW, there is the `withFocusOutside` higher-order component already available today:

https://github.com/WordPress/gutenberg/tree/master/packages/components/src/higher-order/with-focus-outside